packages/bottom-tabs/src/views/BottomTabBar.tsx

```diff
@@ -256,42 +270,67 @@ export function BottomTabBar({
   return (
     <Animated.View
       style={[
-        styles.tabBar,
+        tabBarPosition === 'left'
+          ? styles.left
+          : tabBarPosition === 'right'
+          ? styles.right
+          : styles.bottom,
         {
           backgroundColor:
             tabBarBackgroundElement != null ? 'transparent' : colors.card,
-          borderTopColor: colors.border,
+          borderColor: colors.border,
         },
-        {
-          transform: [
-            {
-              translateY: visible.interpolate({
-                inputRange: [0, 1],
-                outputRange: [
-                  layout.height + paddingBottom + StyleSheet.hairlineWidth,
-                  0,
+        tabBarPosition === 'bottom'
+          ? [
+              {
+                transform: [
+                  {
+                    translateY: visible.interpolate({
+                      inputRange: [0, 1],
+                      outputRange: [
+                        layout.height +
+                          paddingBottom +
+                          StyleSheet.hairlineWidth,
+                        0,
+                      ],
+                    }),
+                  },
                 ],
-              }),
+                // Absolutely position the tab bar so that the content is below it
+                // This is needed to avoid gap at bottom when the tab bar is hidden
+                position: isTabBarHidden ? 'absolute' : (null as any),
```

Comment:
Hm, I'm not sure how you like this pattern, but I typically use

```
   ...(isTabBarHidden && { position: 'hidden' })
```

Reply:
Changed to `position: isTabBarHidden ? 'absolute' : undefined`